README.md

```diff
@@ -25,7 +25,7 @@ $ docker run -it \
 	-v /etc:/rootfs/etc \
 	-v /opt:/rootfs/opt \
 	-v /usr/bin:/rootfs/usr/bin \
-	luxas/kubeadm-installer ${your_os_here}
+	xakra/kubeadm-installer:0.4.5 ${your_os_here}
```

Comment:
I found I got errors unless I had the word "install" included: `xakra/kubeadm-installer:0.4.5 ${your_os_here} install`